apps/beeai-cli/src/beeai_cli/commands/platform.py

```diff
@@ -82,34 +93,60 @@ def _lima_yaml(k3s_port: int = 16443, beeai_port: int = 8333, phoenix_port: int
     hostPort: {phoenix_port}"""
 
 
+@functools.cache
+def _limactl_exe():
+    bundled_limactl = importlib.resources.files("beeai_cli") / "data" / "limactl"
+    return bundled_limactl if bundled_limactl.is_file() else "limactl"
+
+
 def _validate_driver(vm_driver: VMDriver | None) -> VMDriver:
+    is_windows = platform.system() == "Windows" or shutil.which("wsl.exe")
+    has_lima = (importlib.resources.files("beeai_cli") / "data" / "limactl").is_file() or shutil.which("limactl")
+    has_docker = shutil.which("docker")
```

Comment:
Same with docker - maybe a warning that the docker driver is experimental (except on windows)?